Realm/RLMArray.h

```diff
@@ -300,8 +300,3 @@
 
 @end
 
-
-@interface RLMArray (Private)
-- (instancetype)initWithObjectClassName:(NSString *)objectClassName;
```

Comment:
You are both wrong! This is no longer needed as we now import Realm.Private in RLMSupport.swift.